src/tests-tokens/tokens-metadata-tests.ts

```diff
@@ -334,6 +323,17 @@ describe('api tests', () => {
     expect(query1.body.results.length).toStrictEqual(20);
   });
 
+  test('large payload test', () => {
+    //mock the response
+    const rawdata = fs.readFileSync('src/tests-tokens/test-data/large-size-test-data.json');
```

Comment:
We probably don't want to commit large files like this to the repo, especially if we increased the limit to 5MB or something. Can you just generate this data?

Reply:
Neat idea, I think I can.

Reply:
Thanks!